### PR TITLE
Add help formspec for CSM commands

### PR DESCRIPTION
**Pull request by zmv7**
This PR adds graphical interface for `.help` command, the similar as used on the server (`/help`).

## How to test

* Set `enable_client_modding` to `true` in minetest.conf
* Install some client mods (optional)
* Join singleplayer or any server that allows to load client mods
* Type `.help` in the chat and test the functionality of formspec.
* `.help -t` and `.help all -t` should return output to the chat without using formspec

### CODE REVIEW

**Review by appgurueu — CHANGES_REQUESTED** [review]
This looks like it duplicates existing help formspec code. Can't that be moved to `builtin/common/information_formspecs.lua`?

**Comment by zmv7** [comment]
Here we go, now there is universal function for both server and client help formspecs in information_formspecs.lua

File: builtin/common/information_formspecs.lua [hunk 1]
```diff
@@ -86,6 +91,9 @@ end
 -- PRIVILEGES FORMSPEC
 
 local function build_privs_formspec(name)
+	if INIT == "client" then
```
Comment (appgurueu) [inline builtin/common/information_formspecs.lua:94]: Is this change necessary?

**Review by appgurueu — APPROVED** [review]
Looks fine, works.

**Pull request merged**